src/workspace-data/data-table/entity-service/SavedColumnSettings.js

```diff
@@ -358,20 +382,28 @@ export const ColumnSettingsWithSavedColumnSettings = ({ columnSettings, onChange
     div(
       {
         style: {
-          width: '275px',
+          width: '400px',
           paddingLeft: '1rem',
           borderLeft: `1px solid ${colors.light()}`,
           marginLeft: '1rem',
-          marginTop: '2rem',
         },
       },
       [
+        div({ style: { width: 350, marginBottom: '1rem' } }, [
+          h(ConfirmedSearchInput, {
+            'aria-label': 'Search',
+            placeholder: 'Search',
+            onChange: debouncedHandleSearch,
```

Comment:
Can we get rid of the `onChange` now that it's triggering on `onKeyUp`?

Reply:
The search component `ConfirmedSearchInput` I am using needs the `onChange` listener to clear search values when clicking the `x` icon.

<img width="1789" alt="OnChange" src="https://github.com/user-attachments/assets/accd5076-1172-4555-b2af-dd514a2ab35f" />